clang/test/CodeGen/ubsan-trap-reason-add-overflow.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang -O0 -g -debug-info-kind=standalone -dwarf-version=5 -fsanitize=undefined \
```

Comment:
@anthonyhatran We should probably use `%clang_cc1` instead as a lot of codegen tests do that. That by-passes the clang driver (which is responsible for presenting a GCC compatible interface).

We should also hardcode a target triple here so that we get deterministic code generation. Otherwise your test will use the host to determine the target which might cause the tests to fail when running on other host platforms.

E.g.: `%clang_cc1 -triple arm64-apple-macos`